applicationset/controllers/applicationset_controller_test.go

```diff
@@ -2072,36 +2068,8 @@ func TestValidateGeneratedApplications(t *testing.T) {
 			}
 
 			appSetInfo := v1alpha1.ApplicationSet{}
-
 			validationErrors, _ := r.validateGeneratedApplications(context.TODO(), cc.apps, appSetInfo)
-			var errorMessages []string
```

Comment:
You definitely simplified it, so you just started to verify exact error instead some parts of it ?

Reply:
Yeah, the original test for some reason converted the expected error map to a slice and then compared both the map and the slice to the expected values? Just seems way overcomplicated. I changed it to just compare the expected map to the actual map with no further manipulation.